signalfx-forwarder-app/README.md

```diff
@@ -22,6 +22,9 @@ A custom search command to metricize Splunk events and send them to SignalFx
 **tosfx**:       Send the datapoints to SignalFx in a non streaming manner.
                  Does not work in realtime mode, but has higher throughput.
 
+**eventstosfx**:       Send the datapoints to SignalFx as events.
```

Comment:
`tosfxevents` since they are the final format?

Reply:
good point and changed now